src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -1,7 +1,58 @@
 package core.mate.academy.service;
 
-/**
- * Your implementation of MachineService.
- */
-public class MachineServiceImpl {
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MachineServiceImpl implements MachineService<Machine> {
+
+    @Override
+    public List getAll(Class type) {
+        if (type == Bulldozer.class) {
+            BulldozerProducer bulldozerProducer = new BulldozerProducer();
+            return bulldozerProducer.get();
+        } else if (type == Excavator.class) {
+            ExcavatorProducer excavatorProducer = new ExcavatorProducer();
+            return excavatorProducer.get();
+        } else if (type == Truck.class) {
+            TruckProducer truckProducer = new TruckProducer();
+            return truckProducer.get();
+        } else {
+            ArrayList<Machine> machines = new ArrayList<>();
+            return machines;
+        }
+    }
+
+    @Override
+    public void fill(List machines, Object value) {
```

Comment:
same here

Reply:
I don't now what put in line 14 MachineServiceImpl.java file method getAll